addon/templates/components/nypr-accounts/membership-card/help-modal.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if hasBlock}}
+  {{yield}}
```

Comment:
this seems like a pretty specific component. I wonder if this yield is necessary?